content/applications/websites/livechat.rst

```diff
@@ -201,8 +224,8 @@ To add the widget to a website created on a third-party platform, click the firs
 button on the :guilabel:`Widget` tab and paste the code into the `<head>` tag on the site.
 
 Likewise, to send a live chat session to a customer, click the second :guilabel:`COPY` button on the
-:guilabel:`Widget` tab. This link can be sent directly to a customer, and once they click the link,
-it will open a new chat.
+:guilabel:`Widget` tab. This link can be sent directly to a customer. When they click the link, they
+are redirected to a new chat window.
 
 .. image:: livechat/widget-code.png
    :align: center
```

Comment:
might be helpful to put the number 1 and 2 by the links. I skimmed over the numerical labeler you added